dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -194,6 +194,45 @@ public final Response findStepsByScheme(@Context final HttpServletRequest reques
         return response;
     } // findSteps.
 
+    @GET
+    @Path("/contentlet/{inode}/actions")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response findAvailableActions(@Context final HttpServletRequest request,
+                                               @PathParam("inode") final String inode) {
+
+        final InitDataObject initDataObject = this.webResource.init
+                (null, true, request, true, null);
+        Response response;
+        List<WorkflowAction> actions;
+
+        try {
+
+            Logger.debug(this, "Getting the available actions for the contentlet inode: " + inode);
+            actions   = this.workflowHelper.findAvailableActions(inode,
+                            initDataObject.getUser());
+            response  =
+                    Response.ok(new ResponseEntityView(actions)).build(); // 200
+        } catch (DoesNotExistException e) {
+
+            Logger.error(this.getClass(),
+                    "DoesNotExistException on findAvailableActions, contentlet inode: " + inode +
+                            ", exception message: " + e.getMessage(), e);
+            response = ExceptionMapperUtil.createResponse(e, Response.Status.NOT_FOUND);
+        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)

Reply:
We already catch the specific ones